app/layout/project/navigation/hooks.tsx

```diff
@@ -84,19 +86,22 @@ export const useSolutionItems = (): SubMenuItem[] => {
   const { query, route } = useRouter();
   const { pid, sid, tab } = query as { pid: string; sid: string; tab: string };
   const isScenarioRoute = route.startsWith(SCENARIO_ROUTE);
+  const scenarioQuery = useScenario(sid);
 
   return [
     {
       name: 'Overview',
       route: `/projects/${pid}/scenarios/${sid}/edit?tab=${TABS['scenario-solutions']}`,
       icon: OVERVIEW_SVG,
       selected: isScenarioRoute && tab === TABS['scenario-solutions'],
+      disabled: !scenarioQuery.data?.ranAtLeastOnce,
     },
     {
       name: 'Target Achievement',
       route: `/projects/${pid}/scenarios/${sid}/edit?tab=${TABS['scenario-target-achievement']}`,
       icon: TARGET_SVG,
       selected: isScenarioRoute && tab === TABS['scenario-target-achievement'],
+      disabled: !scenarioQuery.data?.ranAtLeastOnce,
```

Comment:
I don't think there is a need to disable the inner options of the menu, disabling the parent (in this case `Solutions`) should be enough as the user won't be able to expand the menu.